### ISSUE TITLE
Webbkartan och menyn hänger inte med statusförändringar

### ISSUE THREAD

**Issue by evajerr** [rinfo/lagrummet.se#362]
Om en redan Publicerad sida med "Visa i Webbkarta" får ett framflyttat datum (tex om man har skrivit fel)och blir "Redo för publicering" så ligger sidan iallafall kvar i menyträdet och i webbkartan.
![a ri fel i publiceras senare](https://cloud.githubusercontent.com/assets/9693918/5919189/b823b1dc-a631-11e4-891d-7d4a5585ee4a.PNG)
![a ri fel i publiceras senare lagrum](https://cloud.githubusercontent.com/assets/9693918/5919191/c00e70bc-a631-11e4-9b5d-62d80a9b8b1f.PNG)

**Comment by jeldeklint** [comment]
det finns inte en äldre version som fortfarande är publicerad?

**Comment by jeldeklint** [comment]
flyttar man _bara_ fram datumet så kommer den gamla versionen nämligen ligga kvar som publicerad. Det går t.ex. utmärkt att ha flera versioner publicerade samtidigt, dock bara den senaste som visas..

**Comment by lalindqv** [comment]
Tänk också på att sidorna kan vara chachade i din browser. 
Tvinga browsern att ladda sidan genon att trycka Ctrl +F5
Jag ska kolla om jag kan återskapa problemet.

**Comment by jeldeklint** [comment]
Mja, tror inte cachen spelar så stor roll här eftersom det inte finns någon cache på sidorna - bara på assets.
Men en annan bugg ni kan testa är att om det finns flera versioner publicerade samtidigt så syns alla versioner i webbkartan - på syskonnivå. Öppnar man sidan visas den senaste dock.

**Comment by lalindqv** [comment]
Problembeskrivningen verkar stämma. Om man flyttar fram Från-datum på en redan publicerad sida till ett framtida datum, så kommer tidigare version att publiceras istället, vilket helt korrekt gör att sidan finns kvar i webbkartan. Det är nog inte webbkartan det är fel på , utan att tidigare versioner ligger i "kö" för publicering.

Det kan alltså vara ganska svårt att i Admin avgöra om en sida är publicerad eller ej.


Sida som är publicerad. Den publicerade sidan visas ej, utan det sparade utkastet som kommer att publiceras senare visas:
![nya sidan](https://cloud.githubusercontent.com/assets/9251555/5935338/28fe8f46-a6da-11e4-94f8-3556f58e59b2.PNG)

Sidan ser ut så här på lagrummet:
![capture](https://cloud.githubusercontent.com/assets/9251555/5935377/c8823572-a6da-11e4-9486-206ef57116f5.PNG)




Sida som inte är publicerad

![tredje sidan](https://cloud.githubusercontent.com/assets/9251555/5935341/39030656-a6da-11e4-845f-9ec9726b6d5a.PNG)

**Comment by kamidev** [comment]
Jag föreslår att vi dokumenterar nuvarande beteende och inte försöker fixa detta just nu.

**Comment by jeldeklint** [comment]
Ja, alltså den tidigare versionen är ju fortfarande publicerad. Det här beteendet är egentligen inget konstigt, även om det kanske inte är helt igenom tänkt hela vägen (att ha flera publicerade versioner _samtidigt_ känns dock mer konstigt).

För att en tidigare publicerad version ska avpubliceras krävs att "publicera till" datumet används. Manuellt eller genom att man trycker på "avpublicera" (knappen sätter "publicera till" datumet till "nu").
